### ISSUE TITLE
Should render call loadData with filters?

### ISSUE THREAD

**Issue by christopher-worley** [tabalinas/jsgrid#191]
I lose my filters after calling render.  I am guessing because render() calls loadData() with no filter.

I've added columns resizing by adjusting the field widths.  Afterwards I call render() to allow the grid to redraw. 

$('#jsGrid').jsGrid("option", "fields")[growIndex].width = growWidth;
$('#jsGrid').jsGrid("render");

This grid has autoload and pageLoading set to true.

Should I call something other than render?

**Comment by soul3lade** [comment]
Hello,
You could try calling refresh instead of render.  That should still redraw the grid but also keep your filters.

`$("#jsGrid").jsGrid("refresh");`

http://js-grid.com/docs/#refresh

**Comment by christopher-worley** [comment]
Thanks for the reply.

Refresh will only resize the jsgrid body.  The header columns stay same size.

**Comment by christopher-worley** [comment]
Looks like the work is done in _prepareCell() which is called for the body cells in refresh; however, it is only called when header rows are created.

For now I'll do the following for my header and filter row.

$('#jsGrid .jsgrid-table .jsgrid-header-row')
  .find('td:eq(1),th:eq(1)')
    .each(function() { $(this).css("width", "50px") });

**Comment by tabalinas** [comment]
Yes, I agree this is unexpected behavior. The `refresh` method should also update `width`, `css`, and `align` of all rows, not only data rows.
